src/main/java/com/sarality/list/NonScrollingListViewInitializer.java

```diff
@@ -39,6 +39,11 @@ public NonScrollingListViewInitializer(Activity activity, LinearLayout listView,
     return this;
   }
 
+  public NonScrollingListViewInitializer<T, H> withEmptyList(View emptyListView) {
```

Comment:
Should I check this in as well. Again breaking the Build because the changes to SubscriptionListItemRenderer are already checked in